composer.json

```diff
@@ -18,8 +18,9 @@
   "require": {
     "php": ">=7.1.0",
     "monolog/monolog": "^1.23.0",
-    "nette/di": "~2.4.12",
-    "nette/utils": "~2.5.2"
+    "nette/di": "~3.0.0",
+    "nette/utils": "~3.0.0",
+    "nette/schema": "~1.0.0"
```

Comment:
nette/schema is part of the nette/di

Reply:
I thinked about the same. But nette/utils also are. Problem is with semantical versioning. nette/di depends on nette/schema^1.0

Reply:
I put nette/schema into composer, because it is used in our code. I know, that it is dependency in nette/di, but I am used to require direct dependencies in composer.

Reply:
I know guys. But it so tied to nette/di so it's not needed. Thanks for removing it. I will merge this PR after that.